pages/[user]/[type].tsx

```diff
@@ -176,8 +189,15 @@ export default function Type(props): Type {
                     disabled={selectedMonth < parseInt(dayjs().format("MM"))}>
                     <ChevronLeftIcon className="w-5 h-5" />
                   </button>
-                  <button onClick={incrementMonth}>
-                    <ChevronRightIcon className="w-5 h-5" />
+                  <button 
+                    onClick={incrementMonth}
+                    className={
+                      minDay !== null &&
+                      selectedMonth >= maxMonth &&
+                        "text-gray-400"
+                    }
+                    disabled={minDay !== null && selectedMonth >= maxMonth}>
+                      <ChevronRightIcon className="w-5 h-5" />
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689307)
```suggestion
                    <ChevronRightIcon className="w-5 h-5" />
```